src/index.html

```diff
@@ -7,6 +7,451 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+            Apple
+            </a>
```

Comment:
```suggestion
            >
              Apple
            </a>
```
each level of nesting should have 2 space indentation